app/src/components/overlay/Modal.tsx

```diff
@@ -142,4 +144,38 @@ function Modal(props: ModalProps, ref: Ref<HTMLDivElement>) {
 }
 
 const _Modal = forwardRef(Modal);
-export { _Modal as Modal };
+
+const modalOverlayCSS = css`
```

Comment:
minor nit, could be easier to find in it's own file

Reply:
I will tackle that in a followup